### PR TITLE
Move R Package to subfolder; Add Python Package Demo

### PR DESCRIPTION
**Pull request by jonyoder**
Connected to rstudio/package-manager#8980. 

* Moves the example R package to another directory. 
* Simplifies the root `README.md` to point to separate readme files for the R and Python examples.
* Creates a new `python-package-manager-demo` directory with its own README and an example Python package.
* Updates the Github Actions workflow to build and install a Python package.

**Note to reviewer**
You can simply verify that the R examples were moved to `r-package-manager-demo/`--nothing significant changed in the R code or description, etc.

### CODE REVIEW

File: .github/workflows/package-manager-demo.yml [hunk 1]
```diff
@@ -72,3 +75,26 @@ jobs:
           PACKAGEMANAGER_TOKEN: ${{ secrets[matrix.package-manager.token] }}
         run: "./rspm add binary --source=local-api --distribution=${{ matrix.os.distro }} --path=$BINARY_FILE || echo 'Already uploaded'"
         shell: bash
+
+      - name: Setup Python 3
```
Comment (tylfin) [inline .github/workflows/package-manager-demo.yml:79]: You might consider making these two separate Github actions for folks that mostly want to copy & paste

File: python-package-manager-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Package Manager Demo
```
Comment (tylfin) [inline python-package-manager-demo/README.md:1]:
```suggestion
# Package Manager Python Package Upload Demo
```

File: r-package-manager-demo/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (tylfin) [inline r-package-manager-demo/LICENSE:1]: You probably want to leave this at the root

File: r-package-manager-demo/README.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
+# Package Manager Demo
```
Comment (tylfin) [inline r-package-manager-demo/README.md:1]:
```suggestion
# Package Manager R Package Upload Demo
```

File: r-package-manager-demo/README.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+[![package-manager-demo](https://github.com/rstudio/package-manager-demo/actions/workflows/package-manager-demo.yml/badge.svg)](https://github.com/rstudio/package-manager-demo/actions/workflows/package-manager-demo.yml)
```
Comment (tylfin) [inline r-package-manager-demo/README.md:3]: Can move the badges to the top level

**Review by tylfin — APPROVED** [review]
Just a few comments, other than that LGTM

File: r-package-manager-demo/README.md (hunk 5 above)
Comment (jonyoder) [reply r-package-manager-demo/README.md:3]: If we have the badges here, then they show up on the README when you load it on RSPM (e.g, look a the readme on solo). Should we leave them here and also have them on the top-level readme?

File: r-package-manager-demo/LICENSE (hunk 3 above)
Comment (jonyoder) [reply r-package-manager-demo/LICENSE:1]: Do we also want the LICENSE included with the packages (e.g., the source, binary, and Python distributions we serve from solo)?

Comment (tylfin) [reply r-package-manager-demo/LICENSE:1]: Oh interesting, I didn't think about the Package Manager side of things... these make sense

File: .github/workflows/package-manager-python-demo.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        r-version: ['3.6.3', '4.1.3']
```
Comment (glin) [inline .github/workflows/package-manager-python-demo.yml:12]: Remove r-version?

File: .github/workflows/package-manager-r-demo.yml [hunk 7]
```diff
@@ -15,9 +15,9 @@ jobs:
           - {os: "ubuntu-18.04", distro: "bionic"}
           - {os: "windows-latest", distro: "windows"}
         package-manager:
-          - {endpoint: "https://solo.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_SOLO"}
-          - {endpoint: "https://s3.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_S3"}
-          - {endpoint: "https://cluster.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_CLUSTER"}
+          - {endpoint: "https://solo.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_SOLO", token_python: "PACKAGEMANAGER_TOKEN_PYTHON_SOLO"}
+          - {endpoint: "https://s3.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_S3", token_python: "PACKAGEMANAGER_TOKEN_PYTHON_S3"}
+          - {endpoint: "https://cluster.rstudiopm.com", token: "PACKAGEMANAGER_TOKEN_CLUSTER", token_python: "PACKAGEMANAGER_TOKEN_PYTHON_CLUSTER"}
```
Comment (glin) [inline .github/workflows/package-manager-r-demo.yml:20]: Remove unused `token_python`?

**Review by glin — APPROVED** [review]
Looks good to me, just noted some unused fields that could be cleaned up.

File: .github/workflows/package-manager-python-demo.yml (hunk 6 above)
Comment (jonyoder) [reply .github/workflows/package-manager-python-demo.yml]: Oh, good catch. I forgot to clean these up when I switched to two separate workflows.

**Pull request merged**